fiftyone/core/dataset.py

```diff
@@ -136,11 +143,22 @@ def load_dataset(name):
 
     Args:
         name: the name of the dataset
+        create_if_necessary (False): if no dataset exists, create an empty one
+
+    Raises:
+        DatasetNotFoundError: if the dataset does not exist and
+            `create_if_necessary` is False
 
     Returns:
         a :class:`Dataset`
     """
-    return Dataset(name, _create=False)
+    try:
+        return Dataset(name, _create=create_if_necessary)
+    except DatasetNotFoundError as ex:
+        if create_if_necessary:
+            return Dataset(name)
```

Comment:
Suggest being explicit with the `_create` argument for clarity's sake.
```suggestion
            return Dataset(name, _create=True)
```